utils/bot.js

```diff
@@ -7,6 +7,7 @@ function partialBotObject(bot) {
     id,
     status: bot.status,
     description: bot.details.shortDescription,
+    notaVerificador: bot.details.notaVerificador ? `${bot.details.notaVerificador}` : 'O usuário não disponabilizou nenhuma nota.' ,
```

Comment:
```suggestion
    notaVerificador: bot.details.notaVerificador || 'O usuário não disponabilizou nenhuma nota.' ,
```

Acredito que isso daria no mesmo resultado
Na verdade seria um problema se o notaVerificador não for uma string, undefined ou null.